### PR TITLE
Switching http url

### PR DESCRIPTION
**Pull request by Gmiller290488**
Made it so the http url depending on platform

### CODE REVIEW

File: androidHackathonApp/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-<string name="heading">CommunityKitchen</string>
+<string name="app_name">CommunityKitchen</string>
```
Comment (Gmiller290488) [inline androidHackathonApp/src/main/res/values/strings.xml:3]: Snuck this change in too 👀

Comment (kirstybutler) [reply androidHackathonApp/src/main/res/values/strings.xml:3]: sneaky

**Review by dsmith42 — APPROVED** [review]
Nice 👍

File: shared/src/commonMain/kotlin/com/kcc/kmmhackathon/shared/network/MealApi.kt [hunk 2]
```diff
@@ -19,32 +24,35 @@ class MealApi {
...
 
-    companion object {
-        // iOS can use this endpoint
-        private const val MEALS_ENDPOINT = "http://localhost:3000/Meals"
+    private fun getEndPoint(): String {
+        if (Platform().isAndroid()) {
+            return localHostAnd + MEALS_ENDPOINT
+        }
+        return localHostiOS + MEALS_ENDPOINT
+    }
```
Comment (kirstybutler) [inline shared/src/commonMain/kotlin/com/kcc/kmmhackathon/shared/network/MealApi.kt:53]: noice

**Review by kirstybutler — APPROVED** [review]

**Review by raynelle03 — APPROVED** [review]

**Pull request merged**